docs/content/developers/design-keuzes.md

```diff
@@ -253,3 +254,27 @@ Een voorbeeld response is dan:
 ```
 
 Merk op dat de concrete codes hier fictief zijn en puur ter illustratie zijn.
+
+# Vertaling van relaties
```

Comment:
M2M is vrij helder op dit moment, over `OneToMany` en `OneToOne` zitten we nog niet allemaal op dezelfde lijn geloof ik :-) We hopen hier vandaag een mep op te geven!

Reply:
@sergei-maertens: wie bedoel je met we zitten nog niet allemaal op dezelfde lijn? Jij en ik? Of jij en anderen? En wie gaan hier vandaag een mep opgeven. Jij en ik? Dat is lastig want ik ben vandaag beperkt beschikbaar. Of jij en anderen?

Reply:
'We' zijn Joeri, Eelco, ik en een aantal andere mensen die dingen met APIs doen op de Fieldlab in Zwolle. We gaan even spieken bij de buren en komen dan hopelijk met een idee over wat er moet gebeuren.

Mijn persoonlijke mening op dit moment is dat we ander typen relaties dan M2M niet per se in de designkeuzes moeten vastleggen tot er een praktische use case voor is, anders vrees ik dat we onszelf gaan vastrijden qua flexibiliteit. Ik denk namelijk dat die andere typen relaties een veel duidelijkere 'richting' hebben en het dus per-resource kan gaan verschillen, en dan heb je dus niet veel aan een 'algemene' design decision.